BingeIT/src/components/VideoDisplay/VideoDisplayVertical.js

```diff
@@ -1,14 +1,24 @@
 import { useVideo } from "../../context/video-context";
 import VideoVertical from "../Videoinfo/vertical/VideoVertical";
 import "./VideoDisplayVertical.css";
+import {
+  searchFilter,
+  categoryFilter,
+} from "../../services/category&searchfilter";
 function VideoDisplayVertical() {
   const { videoState } = useVideo();
-  const { video } = videoState;
+  const { video, category, searchInput } = videoState;
+  const categoryVideo = categoryFilter(category, video);
+  const filterVideo = searchFilter(searchInput, categoryVideo);
   return (
     <div className="video-display-vertical">
-      {video.map((vid) => {
-        return <VideoVertical video={vid} key={vid._id} />;
-      })}
+      {filterVideo.length > 0 ? (
```

Comment:
suggestion : just the fliterVideo.length property will do as the .length propert will return false if its 0 , 
It might help you from next Time onwards

Reply:
yes sure